operator/controllers/seldondeployment_prepackaged_servers.go

```diff
@@ -237,6 +238,41 @@ func (pi *PrePackedInitialiser) addTritonServer(mlDepSpec *machinelearningv1.Sel
 	return nil
 }
 
+func (pi *PrePackedInitialiser) addHuggingFaceDefault(pu *machinelearningv1.PredictiveUnit, deploy *appsv1.Deployment) {
```

Comment:
Would it be simpler to add this one to this file instead: https://github.com/SeldonIO/seldon-core/blob/master/operator/controllers/mlserver.go? 

It should be just a matter of adding the Huggingface server one to this `switch`:

https://github.com/SeldonIO/seldon-core/blob/41738bf547eef2a50a34bd24f5feadd957debbea/operator/controllers/mlserver.go#L234-L250

And then also to this `if` condition:

https://github.com/SeldonIO/seldon-core/blob/41738bf547eef2a50a34bd24f5feadd957debbea/operator/controllers/seldondeployment_prepackaged_servers.go#L423

Reply:
Good shout, yes I agree it would fit better on the mlserver.go file, and adding it to this getmlserverenvvars function

Reply:
This now has been updated - looks much better, let me know if any further thoughts